### ISSUE TITLE
The "i" sign

### ISSUE THREAD

**Issue by cara-tm** [textpattern/textpattern#927]
@philwareham:

The ninth letter "i" for "information" could be changed by an internationally well-known sign, see there:  https://www.toptal.com/designers/htmlarrows/symbols/information-source/

A pretty simple improvement that could be done in an editor that supports 'search and replace' files into a same project.

**Comment by philwareham** [comment]
Yes that sounds totally logical. I do however plan on removing most (if not all) of the links in the panels once we’ve incorporated @makss great work on the help system rewrite. I’ll do this suggestion anyway as it’s quick to do.

**Comment by cara-tm** [comment]
Just found a circled version, too: http://www.amp-what.com/unicode/search/inf

**Comment by philwareham** [comment]
The ℹ symbol is spoken by my screen reader as 'information source' (which is good) whereas the second symbol you mention, ⓘ is not announced at all. Therefore the first option would be the one to use.

**Comment by philwareham** [comment]
Done. Thanks! Closing. I might adjust style at later date if needed.

**Comment by cara-tm** [comment]
;)
